src/PrestaShopBundle/Translation/Loader/SqlTranslationLoader.php

```diff
@@ -50,18 +68,36 @@ public function load($resource, $locale, $domain = 'messages')
             throw new Exception(sprintf('Language not found in database: %s', $locale));
         }
 
-        $translations = Db::getInstance()->executeS('
+        $selectTranslationsQuery = '
             SELECT `key`, `translation`, `domain`
             FROM `'._DB_PREFIX_.'translation`
             WHERE `id_lang` = '.$localeResult['id_lang']
-        );
+        ;
+        $translations = Db::getInstance()->executeS($selectTranslationsQuery);
```

Comment:
it's probably too late, but any chance to drop ``Db`` from any "Symfony" classes ? Maybe using DBAL ?